### ISSUE TITLE
Build failure with XBYAK_NO_EXCEPTION

### ISSUE THREAD

**Issue by nivas-x86** [herumi/xbyak#147]
First of all, thanks for quickly extending the Cpu data structure to 128bits.

But it seems the commit 095ebbf broke the build with XBYAK_NO_EXCEPTION.

Example: Below build runs fine on 3ea8e45
`
make -C sample profiler CXXFLAGS='-DXBYAK_NO_EXCEPTION=1 '
`
But fails with below error on 095ebbf
```
In file included from ../xbyak/xbyak_util.h:21:0,
                 from profiler.cpp:9:
../xbyak/xbyak_util.h: In member function ‘unsigned int Xbyak::util::local::CpuT<dummy>::getNumCores(Xbyak::util::IntelCpuTopologyLevel) const’:
../xbyak/xbyak.h:299:35: error: ‘SetError’ is not a member of ‘Xbyak::util::local’
 #define XBYAK_THROW_RET(err, r) { local::SetError(err); return r; }
                                   ^
../xbyak/xbyak_util.h:295:27: note: in expansion of macro ‘XBYAK_THROW_RET’
   if (!x2APIC_supported_) XBYAK_THROW_RET(ERR_X2APIC_IS_NOT_SUPPORTED, 0)
                           ^
../xbyak/xbyak.h:299:35: note: suggested alternative:
 #define XBYAK_THROW_RET(err, r) { local::SetError(err); return r; }
                                   ^
../xbyak/xbyak_util.h:295:27: note: in expansion of macro ‘XBYAK_THROW_RET’
   if (!x2APIC_supported_) XBYAK_THROW_RET(ERR_X2APIC_IS_NOT_SUPPORTED, 0)
                           ^
../xbyak/xbyak.h:286:13: note:   ‘Xbyak::local::SetError’
 inline void SetError(int err) {
             ^
```

**Comment by herumi** [comment]
Thank you for the report. I've fixed it at v6.051.

**Comment by nivas-x86** [comment]
Thanks for the fix.
